feat: UIG-2774 - form demo uitbreiding

ksvlaanderen opened the pull request:
Playground-lit uitgebreid
Reflect false verwijderd
vl-checkbox-next stories en docs uitgebreid
vl-select-next deletable reactive

[Bamboo](https://www.milieuinfo.be/bamboo/chain/viewChain.action?planKey=UIGOV-CUWC201)
[Jira](https://www.milieuinfo.be/jira/browse/UIG-2774)

Goldflow commented on libs/components/src/next/form/checkbox/stories/vl-checkbox.stories-doc.mdx:
```diff
@@ -22,6 +22,21 @@ import { VlCheckboxComponent } from '@domg-wc/components';
 
 <ArgTypes of={VlCheckboxStories.CheckboxDefault} />
 
+### Met value
```
👍🏼

Goldflow commented on libs/components/src/next/form/checkbox/stories/vl-checkbox.stories-doc.mdx:
```diff
@@ -22,6 +22,21 @@ import { VlCheckboxComponent } from '@domg-wc/components';
+Aangezien readonly er voor zorgt dat de value van een input niet aangepast kan worden, heeft readonly plaatsen op een checkbox geen effect aangezien het aan- en uitvinken de checked state aanpast en niet de value.<br/>
+Als je een checkbox als readonly wilt gebruiken, moet je het `disabled` attribuut meegeven en gebruik maken van een hidden input zodat de value toch mee met het form gesubmit wordt.<br/>
+Zie dat de value van de hidden input overeenkomt met de checked state en de value van de disabled checkbox.
+
```
als ik het goed begrijp is dit dezelfde issue als ze gwn native input checkbox zouden gebruiken; lijkt me niet per sé zo'n courante use-case? maar laat gerust staan hoor - als ze dit dan willen implementeren is er meteen een voorbeeld

ksvlaanderen replied:
Dat is idd hoe je het native ook zou oplossen, heb deze info er bijgezet omdat het afwijkt van hoe de readonly werkt bij de andere componenten.

Goldflow replied:
ok is goed 👍🏼

Goldflow commented on libs/components/src/next/form/checkbox/stories/vl-checkbox.stories-doc.mdx:
```diff
@@ -22,6 +22,21 @@ import { VlCheckboxComponent } from '@domg-wc/components';
+
+## Read only
+
+Aangezien readonly er voor zorgt dat de value van een input niet aangepast kan worden, heeft readonly plaatsen op een checkbox geen effect aangezien het aan- en uitvinken de checked state aanpast en niet de value.<br/>
```
Grammaticaal niets verkeerd, mss vermijden 2 keer "aangezien" in zelfde zin te gebruiken?

ksvlaanderen replied:
Zal het iets beter verwoorden.

Goldflow commented on libs/components/src/next/form/select/vl-select.component.ts:
```diff
@@ -93,19 +93,28 @@ export class VlSelectComponent extends FormControl {
             }
         }
+
+        if (changedProperties.has('deletable')) {
```
mss te laat om op te merken, maar zou verwachten dat `deletable` standaard `true` zou staan (is ook zoals het werkt voor de multiselect by default)

ksvlaanderen replied:
Ik heb dit standaard op false gezet omdat het een variant is, als alle afnemers dat standaard op true zetten kunnen we dat gerust aanpassen.

Goldflow replied:
Ok - that tracks.
Zou te gek zijn om default te veranderen als die op `multiselect` komt te staan.

Goldflow commented:
Goed!

ksvlaanderen committed:
feat: UIG-2774 - form demo uitbreiding

Playground-lit uitgebreid
Reflect false verwijderd
vl-checkbox-next stories en docs uitgebreid
vl-select-next deletable reactive
vl-error-message-next for attribute
